src/display-buffer.coffee

```diff
@@ -860,17 +863,19 @@ class DisplayBuffer extends Model
   # Returns `null` if a wrap wouldn't occur.
   findWrapColumn: (line, softWrapColumn=@getSoftWrapColumn()) ->
```

Comment:
Yeah, let's try moving it into `TokenizedLine` and see how it feels. The method isn't part of the public API.